### PR TITLE
Minor: flex-flow: move comment to content

### PR DESCRIPTION
**Pull request by estelle**
Moved the content from a code comment to readable content

### CODE REVIEW

File: files/en-us/web/css/flex-flow/index.md [hunk 1]
```diff
@@ -61,9 +61,10 @@ See {{cssxref("flex-direction")}} and {{cssxref("flex-wrap")}} for details on th
 
 ### Setting column-reverse and wrap
 
+In this example, the main-axis is the block direction with a reversed main-start and main-end. The flex items are allowed to wrap onto new lines, with the items laid out in multiple lines, if needed.
```
Comment (chrisdavidmills) [inline files/en-us/web/css/flex-flow/index.md:64]:
```suggestion
In this example, the main-axis is the block direction with a reversed main-start and main-end. The flex items are allowed to wrap onto multiple lines if needed.
```

Comment (chrisdavidmills) [reply files/en-us/web/css/flex-flow/index.md:64]: Original was repetitious; I think this can be simpler.

**Review by chrisdavidmills — APPROVED** [review]
@estelle grammar nitpick for you to consider. Approving!

File: files/en-us/web/css/flex-flow/index.md (hunk 1 above)
Comment (estelle) [inline files/en-us/web/css/flex-flow/index.md:64]:
```suggestion
In this example, the main-axis is the block direction with a reversed main-start and main-end. The flex items are allowed to wrap, creating new lines if needed.
```

**Pull request merged**